packages/core/src/components/pagination/pagination.stories.js

```diff
@@ -0,0 +1,327 @@
+import './pagination.css';
+
+export default {
+  title: 'Components/Pagination',
+  decorators: [(storyFn) => `<div style="width: 100%;">${storyFn()}</div>`],
+};
+
+export const Basic = () => `
+<div class="hds-pagination-container">
+  <nav class="hds-pagination" role="navigation" aria-label="Pagination" data-next="Next">
```

Comment:
I'm not sure should we mention in the documentation or in the comments that the data-next attribute is used in styling and it is crucial for the next button to align properly

Reply:
- [ ] Add note about data-next attribute